### PR TITLE
Provide Locational Information for a Manifest with navPlace

### PR DESCRIPTION
**Pull request by thehabes**

### CODE REVIEW

**Comment by thehabes** [comment]
**PLEASE NOTE**
`navPlace` has not yet been published by IIIF Maps TSG.  This PR is to remain on hold until it is published, or at least determined far enough along, to have a recipe represent its use.

**Comment by glenrobson** [comment]
Latest version here: https://preview.iiif.io/cookbook/0154-geo-extension/recipe/0154-geo-extension/

**Comment by glenrobson** [comment]
The cookbook editors reviewed the recipe and are happy for it to go to the TRC with the following small changes:

 * Highlighting don’t need to cover lines 12 to 15 as these are before the navPlace
 * I think we need to turn the validation back on as the manifest is missing IDs on annotation page, annotation and navplace 
 * To keep the manifest as small as possible could we take out one of the manifest labels so we only have one language. Fine to include two labels in the navPlace if you specifically wanted to show how to do two languages. 
 * Include navDate as a related recipe 

Would it be possible to make these changes before the 4th of May so we can send it to the TRC?

Thanks

Glen

**Comment by thehabes** [comment]
@glenrobson I am OK with giving it an id to make it work for now.  However, I disagree that navPlace must have an id to be valid.  It is not a IIIF Resource type and it is not an Annotation.  It is a formatted value, much like the dates for navDate or strings for Text Granularity.  GeoJSON-LD and GeoJSON do not require that GeoJSON data nodes contain an `id`. 

From the GeoJSON Spec
```html
IF a Feature has a commonly used identifier, that identifier SHOULD be included as a member of the Feature object with the name "id", and the value of this member is either a JSON string or number.
```
From the navPlace Extension
```html
A Feature may have an id property.
A Feature Collection may have an id property.
```
So if it always fails validation because there is no id, then you are changing the spec.

**Comment by thehabes** [comment]
@glenrobson I added an `id` to the FeatureCollection and the Feature.  Looks like validation still isn't passing, but because of things in other recipes.  I though maybe it was because I was out of sync with master, but it still failed when I synced up.

**Comment by glenrobson** [comment]
Sorry you are right I miss read the remote FeatureCollection requiring an ID. I'm changing the validator now to make it optional:

https://github.com/IIIF/presentation-validator/pull/137

The reason the build is failing now is not related to the manifest but the recipe needs a topic. So the following in the front matter:

```
topic: geo-recipes
```

**Comment by thehabes** [comment]
Success.  Thank you for the guidance.

File: index.md [hunk 1]
```diff
@@ -147,6 +148,7 @@ _(leading on to segmentation examples later)_
 * Archival collection (hierarchy, paging)
 * Thumbnail range for video navigation
 * [Using Caption and Subtitle Files in Multiple Languages with Video Content][0074]
+* [Represent a Manifest as a Geographic Area in a Web Mapping Client Using navPlace][0154]
```
Comment (glenrobson) [inline index.md:151]: It looks like there are two links to the same recipe

File: index.md [hunk 2]
```diff
@@ -51,6 +51,7 @@ _The corresponding 2.1 test fixture(s) is given like this, where appropriate: ..
 * [Load Manifest Beginning with a Specific Canvas][0202]
 * [Begin playback at a specific point - Time-based media][0015] (65)
 * [Navigation by Chronology][0230]
+* [Display Manifest on a Web Map][0154]
 * [Acknowledge Content Contributors][0234]
```
[the remark this answers is not in the document]
Comment (thehabes) [reply index.md:55]: Good Eye.  I kept the 'Locate a Manifest on a Web Map' under the Chronology, the one closer to the top of the page.

**Comment by thehabes** [comment]
Sorry about that, I think I got one or two of them right at least.

**Comment by glenrobson** [comment]
Cookbook author group reviewed after trc comments and are happy to merge.

**Pull request merged**